ztp/gitops-subscriptions/argocd/example/acmpolicygenerator/hub-side-templating/acm-group-du-sno-ranGen-templated.yaml

```diff
@@ -209,13 +216,14 @@ policies:
           - vfio_pci.disable_idle_d3=1
           - efi=runtime
           cpu:
-            isolated: '{{hub fromConfigMap "" (printf "site-%s-configmap" .ManagedClusterName) (printf "%s-cpu-isolated" (index .ManagedClusterLabels "hardware-type")) hub}}'
-            reserved: '{{hub fromConfigMap "" (printf "site-%s-configmap" .ManagedClusterName)  (printf "%s-cpu-reserved" (index .ManagedClusterLabels "hardware-type")) hub}}'
+            # These must be tailored for the specific hardware platform
+            isolated: '{{hub fromConfigMap "" "group-hardware-types-configmap" (printf "%s-cpu-isolated" (index .ManagedClusterLabels "hardware-type")) hub}}'
```

Comment:
I assume you didn't the namespace as the policy created in the same namespace as the configMap "ztp-group" Is that correct ?